pkgs/tools/misc/woof/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl, python }:
+
+stdenv.mkDerivation rec {
+  version = "2012-05-31";
+  name = "woof-${version}";
+  
+  src = fetchurl {
+    url = "http://www.home.unix-ag.org/simon/woof-${version}.py";
+    sha256 = "d84353d07f768321a1921a67193510bf292cf0213295e8c7689176f32e945572";
+  };
+
+  buildInputs = [ python ];
+  runtimeDeps = [ python ];
```

Comment:
No need for `runtimeDeps`.

Reply:
Actually, I've never seen `runtimeDeps` before.
The manual has a nice overview: https://nixos.org/nixpkgs/manual/#ssec-stdenv-attributes
Essentially, `buildInputs` are for run-time dependencies whereas `nativeBuildInputs` are for build-time dependencies.